src/master/production/sync-replication.md

```diff
@@ -101,17 +101,17 @@ Make sure you already have [ImmuDB installed](../running/download.md).
    Login to immudb
 
    ```shell
-   $ /immuadmin login immudb
+   $ immuadmin login immudb
    ```
 
    Create a database `db` that requires 1 confirmation from the synchronous followers to do the commit. 
 
    > Note that `replication-sync-followers` is not the number of existing followers, but the number of confirmations needed, which ideally should be set to `ceil(number of followers/2)` to commit a transaction on the primary server.
```

Comment:
just general note, if you need note to explain setting, then the setting name is not the right one. maybe it should be replication-sync-quorum or something like that